api/controllers/people.js

```diff
@@ -10,7 +10,7 @@ const getPerson = (id, callback) => {
       message: msg || 'Failed to find person.'
     });
   };
-  db.medic.get(id, (err, doc) => {
+  places.fetchHydratedDoc(id, (err, doc) => {
```

Comment:
This is slower but means the validation passes. If we completely stopped caring about validation we could revert this (and ignore validation)